Makefile

```diff
@@ -6,12 +6,7 @@ REGISTRY?=quay.io/coreos/flannel
 # Default tag and architecture. Can be overridden
 TAG?=$(shell git describe --tags --dirty)
 ARCH?=amd64
-# Only enable CGO (and build the UDP backend) on AMD64
-ifeq ($(ARCH),amd64)
-	CGO_ENABLED=1
-else
-	CGO_ENABLED=0
-endif
+CGO_ENABLED?=0
```

Comment:
Add a comment why we're disabling cgo?

Reply:
Isn't that really the default? It's more we don't need cgo builds anymore if we're not supporting the UDP backend.

Reply:
Yeah, I guess that's fair.